<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerDefinitionDialog</class>
 <widget class="QDialog" name="ServerDefinitionDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection Parameters</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qWolfClient.qrc">
    <normaloff>:/images/world.png</normaloff>:/images/world.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>401</width>
     <height>32</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="nameLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Connection name :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="hostLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Server name / IP :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="portLbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Server port :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="portSpinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>72</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>7661</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hostEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="testBttn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Test Connection</string>
   </property>
  </widget>
  <widget class="QFrame" name="SSLframe">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>401</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="SSLcheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SSL connection</string>
    </property>
   </widget>
   <widget class="QFrame" name="SSLclientFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="certificateLbl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Certificate file :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="certEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="keyLbl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Key file :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="keyEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="certFileBttn">
     <property name="geometry">
      <rect>
       <x>342</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="keyFileBttn">
     <property name="geometry">
      <rect>
       <x>342</x>
       <y>60</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="clientCertCheck">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Send client certificate</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="SSLverifyFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QCheckBox" name="SSLverifyCheck">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Verify Server Certificate</string>
     </property>
    </widget>
    <widget class="QLabel" name="CAbundleLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CA bundle file :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="CAbundleEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="CAbundleFileBttn">
     <property name="geometry">
      <rect>
       <x>342</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QSpinBox" name="timeoutSpinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="suffix">
    <string> sec.</string>
   </property>
   <property name="value">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="timeoutLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Message timeout :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <zorder>SSLframe</zorder>
  <zorder>buttonBox</zorder>
  <zorder>nameLbl</zorder>
  <zorder>hostLbl</zorder>
  <zorder>portLbl</zorder>
  <zorder>portSpinBox</zorder>
  <zorder>nameEdit</zorder>
  <zorder>hostEdit</zorder>
  <zorder>testBttn</zorder>
  <zorder>timeoutSpinBox</zorder>
  <zorder>timeoutLbl</zorder>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>hostEdit</tabstop>
  <tabstop>portSpinBox</tabstop>
  <tabstop>timeoutSpinBox</tabstop>
  <tabstop>SSLcheck</tabstop>
  <tabstop>clientCertCheck</tabstop>
  <tabstop>certEdit</tabstop>
  <tabstop>certFileBttn</tabstop>
  <tabstop>keyEdit</tabstop>
  <tabstop>keyFileBttn</tabstop>
  <tabstop>SSLverifyCheck</tabstop>
  <tabstop>CAbundleEdit</tabstop>
  <tabstop>CAbundleFileBttn</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>testBttn</tabstop>
 </tabstops>
 <resources>
  <include location="qWolfClient.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ServerDefinitionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ServerDefinitionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
